### DISCUSSION TITLE
Introduce fan control

### DISCUSSION THREAD

**Discussion by cmfrazier** [tesla-android/issue-tracker#135] [Ideas]
I think this is a start - https://forum.argon40.com/t/argon-v2-script-for-android/813

**Comment by mikegapinski** [comment]
Fan control using the config.txt file on the boot partition works in Tesla Android already. At least it did a while ago when I tested this

**Reply by sw1002** [reply]
> Fan control using the config.txt file on the boot partition works in Tesla Android already. At least it did a while ago when I tested this

can you give an instruction how to do this with a mac?
